src/test/java/uk/gov/dhsc/htbhf/BrowserStackLauncher.java

```diff
@@ -92,15 +107,18 @@ private static TestExecutionSummary runTest(String testName) {
         launcher.registerTestExecutionListeners(listener);
         launcher.execute(request);
 
-        return listener.getSummary();
+        TestExecutionSummary summary = listener.getSummary();
+        if (!CollectionUtils.isEmpty(summary.getFailures())) {
+            log.error("Test [{}] had failures so triggering a retry.", testName);
```

Comment:
Just to check my understanding, this error will be logged every time the test fails, event when we've reached the maximum number of retries?

Reply:
Yes, that's true, it will also be logged the final time. Not sure I want to pass the counter in so I can check and make this to not display on the final attempt though. I could just remove this log line, what do you think?

Reply:
Have now improved it to only output that log message on all bar the final time which is better.